Assets/MRTK/Core/Definitions/BoundarySystem/MixedRealityBoundaryVisualizationProfile.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.MixedReality.Toolkit.Boundary
     /// <summary>
     /// Configuration profile settings for setting up boundary visualizations.
     /// </summary>
-    [CreateAssetMenu(menuName = "Mixed Reality Toolkit/Profiles/Mixed Reality Boundary Visualization Profile", fileName = "MixedRealityBoundaryVisualizationProfile", order = (int)CreateProfileMenuItemIndices.BoundaryVisualization)]
+    [CreateAssetMenu(menuName = "Mixed Reality/Profiles/Mixed Reality Boundary Visualization Profile", fileName = "MixedRealityBoundaryVisualizationProfile", order = (int)CreateProfileMenuItemIndices.BoundaryVisualization)]
```

Comment:
We should still group these under a "Toolkit" heading, so we don't clutter the top level. Either `Mixed Reality/Toolkit` or `Mixed Reality/Mixed Reality Toolkit` or maybe `Mixed Reality/MRTK`, but it should definitely be grouped imo

Reply:
Sounds reasonable. Please note that making the suggested change will deviate from the top level menu (unless we add a Toolkit or MRTK mid level menu there as well)